.github/workflows/build-images-ci.yaml

```diff
@@ -35,6 +35,7 @@ concurrency:
 jobs:
   build-and-push-prs:
     timeout-minutes: 45
+    name: Build and Push Images
```

Comment:
Should this denote that these are CI images to differentiate from other types of images?